### DISCUSSION TITLE
第1节课笔记(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#31] [笔记]
请在下方提交笔记链接,格式为  
-->报名序号+连接:  
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by Fautines** [comment]
579+https://www.yuque.com/shijiuchennianhua-avnma/pn9l3t/rutw2m7l0tgpyy7n?singleDoc# 《书生·浦语大模型全链路开源体系》

**Comment by DoubleRedX** [comment]
584+https://juejin.cn/spost/7319701574086967307

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/--notebook/blob/main/handWrite01.md

**Comment by yibuxiaoxinjiu** [comment]
524+https://github.com/yibuxiaoxinjiu/InternLm-Code/blob/main/code1.md

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/README.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/InternLM_Notes/blob/main/note1.md

**Comment by 1212jake** [comment]
553+https://github.com/1212jake/-1-8-/blob/main/readme.md

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135386337

**Comment by Alias-z** [comment]
388+http://t.csdnimg.cn/tMMAC

**Comment by 00z768** [comment]
1095+https://github.com/00z768/model/blob/main/书生·浦语大模型全链路开源体系.md

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135394566

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6?theme=light

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/README.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/52889427-06be-4314-8b69-7d1c87f90801

**Comment by L-369-maha** [comment]
521+https://github.com/L-369-maha/InternLM/tree/main/One

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course1.md

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/VopUdVn2Uo80vDxIt1OcN91AnYc?from=from_copylink

**Comment by Zeno673** [comment]
567 + https://github.com/Zeno673/InternLMLearning/blob/main/notes/1.%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E4%BD%93%E7%B3%BB.md

**Comment by Doublefei-Giser** [comment]
409+https://githubfast.com/Doublefei-Giser/free-octo-InternLM/blob/main/README.md

**Comment by Vincent-luo** [comment]
566 + https://www.yuque.com/vincentluo-wbw4o/xpmg2z/vdwkiscxsxnhc8gw?singleDoc# 《笔记1》

**Comment by JieGenius** [comment]
555+ https://github.com/JieGenius/InternLMLearning/blob/main/lesson1/note.md

**Comment by jzh999** [comment]
361+https://www.yuque.com/bright-p9un8/yum3v8/iynkw5z31qmfdgo5?singleDoc# 《第一节、书生·浦语大模型全链路开源体系》

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/Day01.md

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/1note.md
